app/charts/map/map-state.tsx

```diff
@@ -414,14 +417,16 @@ const useNumericalColors = (
 ) => {
   const componentId =
     colorSpec?.type === "numerical" ? colorSpec.componentId : "";
+  const conversionFactor =
+    colorSpec?.type === "numerical"
+      ? (colorSpec.unitConversion?.factor ?? 1)
+      : 1;
   const getValue = useCallback(
     (d: Observation) =>
-      componentId
-        ? d[componentId] !== null
-          ? Number(d[componentId])
-          : null
+      d[componentId] !== null
+        ? Number(d[componentId]) * conversionFactor
```

Comment:
convertUnit function could be used here so that we do not have to do the default to 1 of factor + knowledge of multiply.